api/DockerfileDevMac

```diff
@@ -0,0 +1,19 @@
+## Use a base image that supports Rust and cross-compilation for aarch64
```

Comment:
Can you add a comment explaining why this base image is needed on Darwin for local dev but not for building the int/prod images?